platform/brains2/src/lib/midi_wrapper.cpp

```diff
@@ -0,0 +1,56 @@
+#include "midi_wrapper.h"
+#include "Arduino.h" // For HardwareSerial
+#include <MIDI.h>
+#include <USB-MIDI.h>
+
+static USBMIDI_NAMESPACE::usbMidiTransport usbTransport(0);
+static MIDI_NAMESPACE::MidiInterface<USBMIDI_NAMESPACE::usbMidiTransport>
+    usb_midi(usbTransport);
+
+static MIDI_NAMESPACE::SerialMIDI<HardwareSerial> serialTransport(Serial);
+static MIDI_NAMESPACE::MidiInterface<MIDI_NAMESPACE::SerialMIDI<HardwareSerial>>
+    serial_midi(serialTransport);
+
+#define ALL_TRANSPORTS(function_call)                                          \
```

Comment:
I think having the `MIDI_` prefix here would be superfluous since the whole module is about MIDI, and it's a local macro.